docs/learning/WriteAConnector.md

```diff
@@ -256,13 +256,13 @@ A Connector is usually dealing with two levels of provenance
 1. What is the identity and metadata of a file or repository synchronized into an iModel?
 2. What is the identity of the element within that repository?
 
-[RepositoryLink]($imodeljs-backend) is a specialization of a UrlLink which has a "Url" property pointing to an external resource or repository and a RepositoryGuid that identifies the external repository.
+[RepositoryLink]($core-backend) is a specialization of a UrlLink which has a "Url" property pointing to an external resource or repository and a RepositoryGuid that identifies the external repository.

```

Comment:
These package renames aren't going to work in this branch

Reply:
Just reverted those, as well as the section under "Locks & Codes" which I'm assuming was also describing 3.0 behavior in the version I pushed.